src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -471,8 +471,6 @@ internal TdsParserStateObject GetSession(object owner)
             else
             {
                 session = _physicalStateObj;
-                // But for CoreCLR we are now relying on the owner to ensure that cancellation owner comes from one source
-                session.Owner = owner;
```

Comment:
This line was causing assert failures in the bulk copy tests. Removing it doesn't cause any test failures, and it's not present in the .NET Framework version of TdsParser.

@saurabh500 Thoughts?